azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/WebAppUtils.java

```diff
@@ -161,6 +163,20 @@ public static RuntimeStack getLinuxRunTimeStack(String imageName) throws MojoExe
         throw new MojoExecutionException(IMAGE_NOT_GIVEN);
     }
 
+    public static boolean isUpdateWebAppNecessary(final String schemaVersion, final RuntimeSetting runtime)
```

Comment:
What if the runtime setting is just the same as the actual runtime of the Web App

Reply:
Will still do the update. If there are any improvement space, I think it should be handled in the API side.